specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -981,8 +981,7 @@ Connector architecture must support deploying and using a JDBC driver
 that has been written and packaged as a resource adapter using the
 Connector architecture.
 
-The JDBC 4.2 specification is available at
-_https://jcp.org/en/jsr/detail?id=221_ .
+Every release of Jakarta EE declares a minimum required version of Java SE. For discussion, let this be Java SE N. Compatible implementations of Jakarta EE must support the latest version of the JDBC API mentioned in the Java SE N javadocs for the package `java.sql`. These javadocs typically have a link to the corresponding specification at `jcp.org`. 
```

Comment:
@edburns Hi, does this statement mean that, for example, if the following javadoc is given, compliance with the latest JDBC 4.3 spec becomes a condition for the approval of the platform implementation?
https://docs.oracle.com/en/java/javase/21/docs/api/java.sql/java/sql/package-summary.html